files/build_templates/docker_image_ctl.j2

```diff
@@ -388,13 +390,23 @@ start() {
         $REDIS_MNT \
         -v /usr/share/sonic/device/$PLATFORM:/usr/share/sonic/platform:ro \
 {%- if sonic_asic_platform != "mellanox" %}
+{%- if mount_default_tmpfs is defined and mount_default_tmpfs == "y" %}
         --tmpfs /tmp \
 {%- endif %}
+{%- endif %}
+{%- if mount_default_tmpfs is defined and mount_default_tmpfs == "y" %}
         --tmpfs /var/tmp \
+{%- endif %}
         --env "NAMESPACE_ID"="$DEV" \
         --env "NAMESPACE_PREFIX"="$NAMESPACE_PREFIX" \
         --env "NAMESPACE_COUNT"=$NUM_ASIC \
-        --name=$DOCKERNAME {{docker_image_name}}:latest || {
+        --name=$DOCKERNAME \
+{%- if docker_image_name is defined %}
+        {{docker_image_name}}:latest \
+{%- else %}
+        {{docker_image_id}} \
```

Comment:
Why add docker_image_id? I don't find any clue in your PR description.

Reply:
I updated title and PR description.
With coming app ext feature docker image might not have a 'latest' tag or it might not have a name at all.
So this template is required to accept more general docker_image_id, but preserving compatibility if docker_image_name was passed.